src/main/java/bookstore/validation/Email.java

```diff
@@ -0,0 +1,17 @@
+package bookstore.validation;
+
+import jakarta.validation.Constraint;
+import jakarta.validation.Payload;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Constraint(validatedBy = EmailValidator.class)
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.PARAMETER, ElementType.FIELD})
+public @interface Email {
```

Comment:
Why do you need custom? We have existed jakarta.validation.constraints.Email

Reply:
Didn't know that. Thank you!